lib/screens/send_request/bloc/send_request_bloc.dart

```diff
@@ -28,5 +28,13 @@ class SendRequestBloc extends Bloc<SendRequestEvent, SendRequestState> {
         yield InitialSendRequestState(message: failure.message);
       }
     }
+    if (event is ResetSnackbarMessage) {
+      try {
```

Comment:
try catch is not required here.
yield initialState;  is enough

Reply:
@Gaurav192 done :)